integration-tests/cucumber-tests-platform/src/test/java/org/opensmartgridplatform/cucumber/platform/glue/steps/database/logging/DeviceLogItemSteps.java

```diff
@@ -10,30 +10,63 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.opensmartgridplatform.cucumber.core.ReadSettingsHelper.getString;
 
 import java.util.List;
+import java.util.Map;
 
-import cucumber.api.java.en.Then;
 import org.opensmartgridplatform.cucumber.core.GlueBase;
+import org.opensmartgridplatform.cucumber.platform.PlatformDefaults;
+import org.opensmartgridplatform.cucumber.platform.PlatformKeys;
 import org.opensmartgridplatform.logging.domain.entities.DeviceLogItem;
 import org.opensmartgridplatform.logging.domain.repositories.DeviceLogItemPagingRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
 public class DeviceLogItemSteps extends GlueBase {
```

Comment:
com.alliander.osgp.cucumber.webapps.web.operator.glue.steps.database.WebOperatorDeviceSteps contains a step to generate a set of very basic device log items. Check if we can easily move that method to this class.